authors/apps/authentication/views.py

```diff
@@ -143,3 +146,54 @@ def update(self, request, *args, **kwargs):
         serializer.save()
 
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class PasswordResetView(CreateAPIView):
```

Comment:
Can't a `patch` in the `UserRetrieveUpdateAPIView` since a password update is just a partial update to a user's information, it doesn't require an entire view to itself.